sajibportfolio.css

```diff
@@ -4,6 +4,10 @@ body {
   margin: 0;
 }
 
+img[alt] {
+  padding: 0.8rem;
+}
+
```

Comment:
[Optional] I think that to style the alt text, targeting the `img` tag will do the job instead. The defined rule might not match any element.

Reply:
ok, changed it to img only. Thanks